com.unity.render-pipelines.core/Runtime/Unity.RenderPipelines.Core.Runtime.asmdef

```diff
@@ -25,7 +25,12 @@
             "name": "com.unity.inputsystem",
             "expression": "0.0.0",
             "define": "ENABLE_INPUT_SYSTEM_PACKAGE"
+        },
+        {
+            "name": "com.unity.rendering.hybrid",
+            "expression": "0.6.9",
```

Comment:
What was the reason for bumping this down to 0.6.9? This translates to `>= 0.6.9` according to [the docs.](https://docs.microsoft.com/en-us/nuget/concepts/package-versioning#version-ranges-and-wildcards)

Reply:
Are you sure? I tested this locally and it took the wrong path if I put it at 0.7.0 here. X.Y.9 is used in a lot of other places as well.

In any case: I will move this to be 0.6.0 since that is the version the dots side will happen on. I will test locally again to see if 0.6.0 works or if I have to go "one below"